### PR TITLE
minor typos / QoL / updated patches

### PR DESCRIPTION
**Pull request by emberian**
minor things from https://github.com/MinaProtocol/mina/pull/15429

### CODE REVIEW

**Review by ghost-not-in-the-shell — COMMENTED** [review]
The changes to log level and log format looks good to me

**Review by ghost-not-in-the-shell — APPROVED** [review]

**Comment by deepthiskumar** [comment]
!ci-build-me

**Review by georgeee — COMMENTED** [review]
Could you elaborate on what have changed in patches?

File: buildkite/scripts/export-git-env-vars.sh [hunk 1]
```diff
@@ -59,6 +59,8 @@ else
               RELEASE=berkeley ;;
             *rampup*) # any tag including the string `rampup`
               RELEASE=rampup ;;
+            *devnet-tooling*)
```
Comment (amc-ie) [inline buildkite/scripts/export-git-env-vars.sh:62]: Can we change this to devnet to make it generic for our subsequent releases?

**Comment by deepthiskumar** [comment]
!ci-nightly-me

File: scripts/archive/migration/mina-berkeley-migration-script [hunk 2]
```diff
@@ -744,8 +743,8 @@ function run_final_migration() {
         --archive-uri "$__migrated_archive_uri" \
         --input-file "$__replayer_checkpoint" \
         --checkpoint-interval "$__checkpoint_interval" \
-        --log-json \
-        --checkpoint-file-prefix "$CHECKPOINT_PREFIX" |& tee "$__replayer_log" | mina-logproc
+        --checkpoint-file-prefix "$CHECKPOINT_PREFIX" \
+        --log-files "$__replayer_log"
```
[the remark this answers is not in the document]
Comment (emberian) [reply scripts/archive/migration/mina-berkeley-migration-script:747]: pls use 
```suggestion
        --log-file "$__replayer_log"
```
button ty

**Comment by deepthiskumar** [comment]
!ci-nightly-me

**Comment by deepthiskumar** [comment]
!ci-build-me

**Review by amc-ie — APPROVED** [review]
Approved from a CI pov, thanks!

**Review by georgeee — APPROVED** [review]

**Review by SanabriaRusso — APPROVED** [review]

**Comment by deepthiskumar** [comment]
develop merge conflicts from existing state of berkeley. Will follow up on that separately.

**Pull request merged**